apps/console/src/features/oidc-scopes/components/wizards/add-oidc-scope-wizard.tsx

```diff
@@ -81,19 +82,19 @@ export const OIDCScopeCreateWizard: FunctionComponent<OIDCScopeCreateWizardProps
     const [ wizardState, setWizardState ] = useState<WizardStateInterface>(undefined);
 
     const [ OIDCAttributes, setOIDCAttributes ] = useState<ExternalClaim[]>(undefined);
-    const [ selectedAttributes, setSelectedAttributes ] = useState<ExternalClaim[]>([]);
-    const [ isClaimRequestLoading, setIsClaimRequestLoading ] = useState<boolean>(false);
+    const [ selectedAttributes ] = useState<ExternalClaim[]>([]);
+    const [ , setIsClaimRequestLoading ] = useState<boolean>(false);
```

Comment:
Seems like these state variables are fully utilized in the code. therefore, shall we remove these?

Reply:
Removed with 8e57112872a5c3ba2466bdc7038c626693f0657b.